server/utils/characteristicsUtils.ts

```diff
@@ -1,6 +1,27 @@
-import { CharacteristicPair } from '@approved-premises/api'
-import characteristicLookup from '../i18n/en/characteristics.json'
+import { Cas1SpaceBookingCharacteristic, CharacteristicPair } from '@approved-premises/api'
```

Comment:
Should these utilities refer to the `Cas1SpaceBookingCharacteristic` type? It seems the more generic `Cas1SpaceCharacteristic` would be more suited -- although this might only move some of the type casting around...

Reply:
Yes, you're right. Cas1SpaceBookingCharacteristic is right for the map though as that only maps the names for the 6 room-level ones.
The switch didn't throw up any type errors.